code/modules/admin/verbs/artifacts.dm

```diff
@@ -35,6 +35,8 @@
 			"ref" = REF(art),
 			"loc" = "[AREACOORD(art)]",
 			"active" = component.active,
+			"trigger" = component.activators,
+			"effect" = component.chosen_fault,
```

Comment:
you should really convert these to lists of strings and pass them, instead of relying on their names being turned to strings